### PR TITLE
Flow Control - Editing count by entries always edit the first entry as well

### PR DESCRIPTION
**Pull request by denis-reblaze**

### CODE REVIEW

File: curiefense/ui/src/components/__tests__/LimitOption.spec.ts [hunk 1]
```diff
@@ -5,8 +5,11 @@ import {mount, Wrapper} from '@vue/test-utils'
 import Vue from 'vue'
 
 describe('LimitOption.vue', () => {
+
   let option: OptionObject
   let wrapper: Wrapper<Vue>
+  const WANTED_INDEX: Number = Math.floor ( Math.random() * 10000 )
```
Comment (Aviv-Galmidi) [inline curiefense/ui/src/components/__tests__/LimitOption.spec.ts]: Please keep naming to the typescript convention - lowerCamelCase

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: curiefense/ui/src/components/__tests__/LimitOption.spec.ts (hunk 1 above)
Comment (denis-reblaze) [reply curiefense/ui/src/components/__tests__/LimitOption.spec.ts]: typescript convention for the constants is CONSTANT_CASE https://google.github.io/styleguide/tsguide.html

**Comment by Aviv-Galmidi** [comment]
Let's add a test case which tests this bug - when there are multiple count by keys in flow control and we change one of them, assert that it changed and no other count by keys have changed

File: curiefense/ui/src/doc-editors/__tests__/FlowControlEditor.spec.ts [hunk 2]
```diff
@@ -11,53 +11,52 @@ import axios from 'axios'
...
-  beforeEach(() => {
-    docs = [
-      {
-        'exclude': ['devops', 'internal'],
-        'include': ['china'],
-        'name': 'flow control',
-        'key': [
-          {
-            'attrs': 'ip',
+  const docs: FlowControl[] = [
```
Comment (Aviv-Galmidi) [inline curiefense/ui/src/doc-editors/__tests__/FlowControlEditor.spec.ts:15]: This pattern is intentional as we might mutate the 'docs' variable within the tests. As we don't want tests to leak and influence one another, we're resetting docs to an initial state before each test run

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

**Review by Aviv-Galmidi — DISMISSED** [review]

**Comment by flaper87** [comment]
@denis-reblaze This needs rebase, any chance you can take care of it?

**Comment by denis-reblaze** [comment]
I 'll create another pr for this feature since this pr is incorrect (accidentally made on main branch)

**Pull request closed without merging**